.github/workflows/build.yml

```diff
@@ -15,14 +15,18 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: self-hosted
+    env:
+      KUBECONFIG: /etc/rancher/k3s/k3s.yaml
```

Comment:
how does it know where our running is? and where does it get this file from?

Reply:
the GH agent is running on the host and is automagically connecting to GH so I can't say anything beyond that